File: scheduling-service/internal/domain/appointment.go
package domain

type Appointment struct {
	Id          int    `json:"id"`
	Description string `json:"description" binding:"required"`
	DateAndTime string `json:"dateAndTime" binding:"required"`
	DentistCRO  string `json:"dentistCRO" binding:"required"`
	PatientRG   string `json:"patientRG" binding:"required"`
}

File: invoice-service/src/main/java/com/ronilsonalves/invoiceservice/data/dto/Appointment.java
package com.ronilsonalves.invoiceservice.data.dto;

import org.springframework.data.jpa.repository.Temporal;
import org.springframework.format.annotation.DateTimeFormat;

import java.time.LocalDateTime;

public record Appointment (
        Integer id,
        String description,
        LocalDateTime dateAndTime,
        String dentistCRO,
        String patientRG,
        Dentist dentist,
        Patient patient
) {

}